docs/working_docs/usage.md

```diff
@@ -95,6 +95,17 @@ it should infer the parser based on the filename:
 osv-scanner --lockfile ':/path/to/my:projects/package-lock.json'
 ~~~~~~~~~~~~~~~~~~~
 
+### Scanning with call analysis (preview)
```

Comment:
Should we add a line to our docs on SemVer adherence that we can break `--experimental` flags and `experimental-` fields in our JSON at any time?

Reply:
Added a highlighted note here and in the installation.md part where we explain our semver adherence